style.css

```diff
@@ -363,3 +357,92 @@ label {
   line-height: 32px;
   color: #42526e;
 }
+
+/* desktop version //////////////////// */
+
+@media screen and (min-width: 768px) {
+  body {
+    width: 1440px;
+  }
+
+  /* headline /////////////////////////// */
+
+  .app-bar-and-headline-container {
+    background-image: url(images/backgrounds/desktop-header-llustration.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+  }
+
+  .toolbar {
+    display: none;
+  }
+
+  .toolbar-desktop {
+    display: flex;
+    justify-content: center;
+    column-gap: 50px;
+  }
+
+  .headline {
+    width: 920px;
+    background-image: none;
+    margin: 0 auto;
+  }
+
+  /* recent-works  /////////////////////// */
+
+  .recent-works .text {
+    padding: 100px 0 40px 0;
+  }
+
+  .indicator {
+    display: none;
+  }
+
+  .card-work-row {
+    grid-template-columns: repeat(3, 1fr); /* 3 columns with equal width */
+    grid-template-rows: repeat(2, 1fr);
+    column-gap: 24px;
+    padding: 0 134px;
+  }
+
+  /* about me /////////////////////////// */
+
+  .about-me {
+    padding: 60px 134px;
+  }
+
+  .about-text {
+    padding: 0 122px;
+  }
+
+  .supporting-text-1 {
+    padding: 0 70px;
+  }
+
+  .about-cards {
+    grid-template-columns: repeat(3, 1fr); /* 3 columns with equal width */
+    column-gap: 24px;
+  }
+
+  .about-me .about-text {
+    padding-bottom: 50px;
+  }
+
+  /* CONTACT-FORM /////////////////////// */
+
+  .contact-form {
```

Comment:
- [optional] please this is only optional because it was not mentioned but is will make your work look great. ensure that your form submit button follows the position of the figma version
- also, your form should align with the cards up and should not cover the background image on the bottom right. 

yours
 
![Screenshot (817)_LI](https://user-images.githubusercontent.com/65298048/220776553-fae87372-a9b5-4ed0-9872-51c18f1529bd.jpg)
figma design
![Screenshot (818)](https://user-images.githubusercontent.com/65298048/220776642-67a9331c-1b48-4cd4-987b-16cbaa0b05fd.png)